src/bootstrap/bootstrap.py

```diff
@@ -201,8 +201,8 @@ def download_stage0(self):
         if self.cargo().startswith(self.bin_root()) and \
                 (not os.path.exists(self.cargo()) or self.cargo_out_of_date()):
             self.print_what_it_means_to_bootstrap()
-            filename = "cargo-{}-{}.tar.gz".format(channel, self.build)
-            url = "https://static.rust-lang.org/dist/" + self.stage0_rustc_date()
+            filename = "cargo-{}-{}.tar.gz".format('0.18.0', self.build)
```

Comment:
Note that this is a bit of an unfortunate hack, I'll try to clean it up on the master branch so we don't have to do this again. This version number should be configured in `src/stage0.txt`